src/defaultV2-theme/blocks/sd-rating.scss

```diff
@@ -69,6 +69,103 @@ legend+sv-ng-rating-item+.sd-rating__item {
   box-shadow: inset 0 0 0 2px $background;
 }
 
+.sd-rating__item-star {
+  position: relative;
+  background: $background;
+  width: calcSize(6);
+  height: calcSize(6);
+  &:not(:first-of-type) {
+      padding-left: calcSize(0.5);
+      margin-left: calcSize(-0.5);
+    }
+  
+    &:not(:last-of-type) {
+      padding-right: calcSize(0.5);
+      margin-right: calcSize(-0.5);
+    }
+
+  svg {
+    stroke: $border;
+    fill: none;
+  }
+
+  .sv-star-2 {
+    display: none;
+  }
+}
+
+.sd-rating__item-star--selected {
+  svg {
+    stroke: $primary;
+    fill: $primary;
+  }
+}
+
+.sd-rating__item-star--error {
+  svg {
+    stroke: $red-light;
+    fill: $red-light;
+  }
+}
+
+.sd-rating__item-star.sd-rating__item-star--disabled {
```

Comment:
May be we could do similar selectors shorter - instead of .sd-rating__item-star.sd-rating__item-star--disabled left modificator only - .sd-rating__item-star--disabled